core/store/migrate/migrations/0227_dynamic_pipeline_runs.sql

```diff
@@ -0,0 +1,34 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE public.job_pipeline_specs (
```

Comment:
@vyzaldysanchez You probably want to drop the `public` here? Looking at other migrations, I don't see that we ever add it so we must be falling back to the default. (The notable exception is `evm` tables.)